spec/change_set_persisters/change_set_persister/cleanup_terms_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ChangeSetPersister::CleanupTerms do
```

Comment:
Interesting, we needed to add this test because we can't exercise the code any other way? Is it still useful to have the cleanup terms even if it never executes? Do we think it will execute in some circumstance, like if we force something via console?

Reply:
That was my thought. It can be deleted in the console and we should still ensure that the ids are cleaned up.